### PR TITLE
Add ios all in one page.

### PR DESCRIPTION
**Pull request by jthoms1**

### CODE REVIEW

File: website/docs/getting-started/iOS-all-in-one.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+# iOS Quick Start
+
+This is a walkthrough on how to get a single Portal + web application setup. To begin this walkthrough it will be assumed that you have already signed up for access to Portals and you have your Portals key ready.
+
+First have your web application ready. We will add some configuration to it and then get it setup in Appflow.
+
+## 1. Create a capacitor config
+
+Create a `capacitor.config.json` file to the root of your web project. We will not need to add Capacitor to the web project but this configuration file will help us configure some options for the Portals that is displaying this web application.
```
Comment (giralte-ionic) [inline website/docs/getting-started/iOS-all-in-one.md:18]: fix grammar ' ...options for the Portal that **are** displaying...'

**Review by carlpoole — APPROVED** [review]

File: website/docs/getting-started/iOS-all-in-one.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+```
+
+### Create the first Portal
+
+Now that the application has been configured all we need to do is add the Portal to a view. By default, a `Portal` will use the `name` property as the directory to load web content from (relative to the root of `Bundle.main`). You can also specify another directory for the content.
+
+```swift
+class ViewController: UIViewController {
+    override func loadView() {
+        self.view = PortalUIView(portal: "featured_products)
```
Comment (giralte-ionic) [inline website/docs/getting-started/iOS-all-in-one.md:211]: missing closing quote

**Review by giralte-ionic — COMMENTED** [review]
I love the format, just a couple corrections needed

File: website/docs/getting-started/iOS-all-in-one.md (hunk 1 above)
Comment (jthoms1) [reply website/docs/getting-started/iOS-all-in-one.md]: Fixed

File: website/docs/getting-started/iOS-all-in-one.md (hunk 2 above)
Comment (jthoms1) [reply website/docs/getting-started/iOS-all-in-one.md]: Fixed

**Review by giralte-ionic — APPROVED** [review]

**Pull request merged**